scripts/vpn-killswitch.sh

```diff
@@ -16,7 +16,9 @@ readonly SCRIPT_DIR="$(dirname "$SCRIPT_PATH")"
 readonly SCRIPT_CONFIG="$SCRIPT_DIR/$SCRIPT_NAME.conf"
 readonly SCRIPT_TAG="$(basename "$SCRIPT_PATH")"
 
-BRIDGE_INTERFACE="br+" # the bridge interface to set rules for, by default affects all "br" interfaces (which also includes guest network bridge)
+# `nvram get apg_ifnames` command will list you the current bridge interfaces excluding br0 in case you need to
+# selectively pick one or more of the bridge interfaces
+FOR_BRIDGE_INTERFACES="br+" # the bridge interface to set rules for, by default affects all "br" interfaces (which also includes guest network bridge)
```

Comment:
`FOR_` prefix in this variable is not needed, `BRIDGE_INTERFACES` is enough
I think you based off this change on `FOR_IPTABLES` variable

It would also be nice to add `, separated by spaces` to the comment on this variable, like I've done here: https://github.com/jacklul/asuswrt-scripts/blob/master/scripts/force-dns.sh#L31

Reply:
> I think you based off this change on `FOR_IPTABLES` variable

Exactly and sure I rebased the PR and removed it. Good point about updating the comment 👍 added.